projects/media-viewer/src/lib/media-viewer.component.ts

```diff
@@ -90,14 +94,14 @@ export class MediaViewerComponent implements OnChanges, OnDestroy, AfterContentI
       this.toolbarEvents.reset();
       this.commentService.resetCommentSet();
       if (this.enableAnnotations) {
-        this.annotationSet = this.api.getAnnotationSet(this.url);
+        this.store.dispatch(new fromStore.LoadAnnotationSet(this.url));
       }
       if (this.contentType === 'image') {
         this.documentTitle = null;
       }
     }
     if (changes.enableAnnotations && this.enableAnnotations) {
-      this.annotationSet = this.api.getAnnotationSet(this.url);
+      this.store.dispatch(new fromStore.LoadAnnotationSet(this.url));
```

Comment:
not sure if this is needed anymore, because we're binding to the annotationSet in the html.
this was needed when we were doing things dynamically.
obviously, you can test it to double check

Reply:
is needed